lightningd/peer_control.c

```diff
@@ -2522,7 +2522,7 @@ static struct command_result *json_sendcustommsg(struct command *cmd,
 		    type, peer_wire_name(type));
 	}
 
-	if (type % 2 == 0) {
+	if (type % 2 == 0 && !cmd->ld->config.allow_even_custom_messages) {
 		return command_fail(
 		    cmd, JSONRPC2_INVALID_REQUEST,
 		    "Cannot send even-typed %d custom message. Currently "
```

Comment:
Suggest also expanding this message to include "If you really want to send even custom messages, add the `--experimental-even-onion-messages` to the `lightningd` configuration"

Reply:
done